tests/qinfo/test_entropies.py

```diff
@@ -81,16 +81,19 @@ class TestVonNeumannEntropy:
     parameters = np.linspace(0, 2 * np.pi, 20)
     devices = ["default.qubit", "default.mixed"]
 
+    diff_methods = ["backprop", "finite-diff"]
+
     @pytest.mark.parametrize("wires", single_wires_list)
     @pytest.mark.parametrize("param", parameters)
     @pytest.mark.parametrize("device", devices)
     @pytest.mark.parametrize("base", base)
-    def test_IsingXX_qnode_entropy(self, param, wires, device, base):
+    @pytest.mark.parametrize("diff_method", diff_methods)
+    def test_IsingXX_qnode_entropy(self, param, wires, device, base, diff_method):
```

Comment:
This is not the right file to change, here we are testing the `qml.qinfo.vn_entropy`, it should be changed in `tests/math/test_entropies_math.py`

Reply:
Previously I had the tests for `qml.mutual_info` already in the `tests/qinfo/test_entropies.py` file though, because I thought the `tests/math/test_entropies_math.py` file was only for the `qml.math` functions (whereas `qml.mutual_info` is a measurement process). So the tests are split up differently between files for the `qml.vn_entropy` and `qml.mutual_info` measurements.

Maybe we can clean up the tests so they are consistent in a separate PR? I'll still move the `vn_entropy` tests to `tests/math/test_entropies_math.py`.

Reply:
OK, should be good to go then. I changed `tests/qinfo/test_entropies.py` for the `qml.mutual_info` measurement and changed `tests/math/test_entropies_math.py` for the `qml.vn_entropy` measurement.